src/permissions.js

```diff
@@ -2,12 +2,23 @@ import memoize from 'lodash.memoize'
 
 const ANY_ADDRESS = '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF'
 
-const APP_MANAGER_ROLE = {
-  name: 'Manage apps',
-  id: 'APP_MANAGER',
-  params: [],
-  bytes: '0xb6d92708f3d4817afc106147d969e229ced5c46e65e0a5002a0d391287762bd0',
-}
+const KERNEL_ROLES = [
+  {
+    name: 'Manage apps',
+    id: 'APP_MANAGER',
+    params: [],
+    bytes: '0xb6d92708f3d4817afc106147d969e229ced5c46e65e0a5002a0d391287762bd0',
+  },
+  {
+    name: 'Create permissions',
+    id: 'CREATE_PERMISSIONS',
+    params: [],
+    bytes: '0x0b719b33c83b8e5d300c521cb8b54ae9bd933996a14bef8c2f4e0285d2d2400a',
+  },
```

Comment:
I think for our sanity later, it would be good to put `CREATE_PERMISSIONS` in an `ACL_ROLES` variable (and then we can concat them, if that's easier), as it actually only exists on the ACL.

Reply:
Done